### ISSUE TITLE
accessibility issue with screen reader

### ISSUE THREAD

**Issue by lstar4ever** [Woundorf/foxreplace#249]
Hi,
Thanks for the addons.
I am a screen reader user and has come across accessibility issue when using the firefox addons.
1. problem with the Foxreplace option page.
there are editable text field according to my screen reader, namely enable, name, etc those 7 edit box though it is editable box acccording to screen reader, typing any text does not have any change

2 problem with the add new list
If I press the add button to create a new list,
options of Mode (Auto & manual) and html (no) cannot be modified with screen reader key press;
cannot add new substitutions to the list as interacting with screen reader has significant problem;
 there are few buttons at the bottom above ok, cancel which have no labels and screen reader only read 
as button which don't know what they are for;
 the ok, cancel and apply button can not be navigated via tab  

---
well, I cannot proceed after the add new list process so I am not sure if there are still further problem, but I am very eager to test out if there will be any improvements.

**Comment by Woundorf** [comment]
Hi, thank you for reporting this. I have tried enabling the screen reader from GNOME to see what happens but it doesn't read anything inside Firefox.

Anyway, the first problem is "normal" because the only editable thing in that table are the checkboxes in the first column (Enable), the others are only to display information and the last one to contain the buttons.

On the second problem, the mode and html options can be changed with the keyboard, so I don't know what happens with the screen reader, I'll have to investigate. The buttons without labels that you say have only icons with text in tooltips. These will be the easiest to fix. Also in this add list dialog there are two tabs, that now that I have tried I can't find how to navigate them with the keyboard, so I suppose they may also be a problem.

I will look for a working screen reader to observe these problems and try to fix them in the following releases.

Thank you again for the report.
